tests/Unit/PointwiseFunctions/AnalyticSolutions/FirstOrderEllipticSolutionsTestHelpers.hpp

```diff
@@ -85,11 +91,114 @@ void verify_solution(
       inertial_coords,
       db::wrap_tags_in<::Tags::FixedSource, typename System::primal_fields>{}));
 
-  Approx numerical_approx = Approx::custom().epsilon(tolerance).scale(1.);
-  CHECK_VARIABLES_CUSTOM_APPROX(
-      get<operator_applied_to_fields_tag>(box),
-      db::item_type<operator_applied_to_fields_tag>(fixed_sources),
-      numerical_approx);
+  // Check error norms against the given tolerance
+  const auto& operator_applied_to_fields =
+      get<operator_applied_to_fields_tag>(box);
+  tmpl::for_each<db::get_variables_tags_list<fields_tag>>(
+      [&operator_applied_to_fields, &fixed_sources,
+       &tolerance](auto field_tag_v) {
+        using field_tag = tmpl::type_from<decltype(field_tag_v)>;
+        const auto& operator_applied_to_field =
+            get<Tags::OperatorAppliedTo<field_tag>>(operator_applied_to_fields);
+        const auto& fixed_source =
+            get<::Tags::FixedSource<field_tag>>(fixed_sources);
+        double l2_error_square = 0.;
+        double linf_error = 0.;
+        for (size_t i = 0; i < operator_applied_to_field.size(); ++i) {
+          const auto error = operator_applied_to_field[i] - fixed_source[i];
+          l2_error_square += alg::accumulate(square(error), 0.0) / error.size();
+          const double component_linf_error =
+              *std::max_element(error.begin(), error.end());
+          if (component_linf_error > linf_error) {
```

Comment:
You might need to take `abs(error)` somewhere before making this comparison.

Reply:
good catch, thanks! squashed this in as well